task/generate-odcs-compose/0.1/generate-odcs-compose.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: generate-odcs-compose
+spec:
+  params:
+    - name: IMAGE
+      type: string
+      description: The image to use for this task
+    - name: KT_PATH
```

Comment:
From where does it take krb5 config?

Or maybe I just miss little bit of more context how krb5 client should be configured.

Reply:
I'm getting lost here.

So you have keytab and you can use keytab to acquire krb ticket, so why do you need cronjob?

Reply:
okay, I'm worried to see krb in such workloads, but we will see

Reply:
Yes. Unfortunately, only kerberos is supported for authentication with ODCS at the moment.